src/main/java/org/broadinstitute/hellbender/utils/read/GATKRead.java

```diff
@@ -73,6 +73,21 @@ default boolean isEmpty() {
     void setPosition( final Locatable locatable );
 
     /**
+     * @return The actual contig assigned to the read, regardless of unmapped status. Unlike {@link #getContig}, will
```

Comment:
Improved the wording, and clarified that the return value is dependent on the underlying read implementation.